### PR TITLE
Feat/461201  add scopes to endpoints

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/routes/activities.js [hunk 1]
```diff
@@ -108,6 +111,7 @@ module.exports = [
     path: '/activity',
     options: {
       tags: ['api'],
+      auth: { scope: [scopes.admin] },
```
Comment (jbarnsley10) [inline app/routes/activities.js:114]: Is this not 'Add an activivty' which is available to standard users?

File: app/routes/courts.js [hunk 2]
```diff
@@ -30,6 +31,7 @@ module.exports = [
     method: 'POST',
     path: '/courts',
     options: {
+      auth: { scope: [scopes.admin] },
       tags: ['api'],
       notes: ['Creates a new court'],
       response: {
```
Comment (jbarnsley10) [inline app/routes/courts.js:37]: Need to add scope to DELETE method of courts too

File: app/routes/export.js [hunk 3]
```diff
@@ -26,6 +28,7 @@ module.exports = [{
   method: 'GET',
   path: '/export-create-file',
   options: {
+    auth: { scope: [scopes.admin] },
```
Comment (jbarnsley10) [inline app/routes/export.js:31]: We need to test this withe the internal loopback overnight job call

File: app/routes/user.js [hunk 4]
```diff
@@ -113,6 +113,7 @@ module.exports = [
     method: 'DELETE',
     path: '/user/{userId?}',
     options: {
+      auth: { scope: [scopes.admin] },
```
Comment (jbarnsley10) [inline app/routes/user.js:116]: I think we can add 'POST /user' and 'POST /users' under admin,

**Review by jbarnsley10 — CHANGES_REQUESTED** [review]
Looks good. A couple of minor changes

File: app/routes/user.js (hunk 4 above)
Comment (whitewaterdesign) [reply app/routes/user.js:116]: these were done previously, a week or so ago

File: app/routes/activities.js (hunk 1 above)
Comment (whitewaterdesign) [reply app/routes/activities.js:114]: Good call, the admin endpoint is `activities`

File: app/routes/courts.js (hunk 2 above)
Comment (whitewaterdesign) [reply app/routes/courts.js:37]: Good catch, thanks

**Review by jbarnsley10 — APPROVED** [review]
LGTM

**Pull request merged**